definitions/divorce/json/CaseEvent.json

```diff
@@ -1990,5 +1990,18 @@
     "PreConditionState(s)": "*",
     "PostConditionState": "AosAwaitingSol",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "16/05/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "grantDnMakeDecision",
+    "Name": "Make a Decision",
+    "Description": "Make a Decision",
+    "DisplayOrder": 3,
+    "PreConditionState(s)": "AwaitingConsideration",
+    "PostConditionState": "*",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/dn-about-to-be-granted",
```

Comment:
Does this endppint exist?

Reply:
It will when this is merged:
https://github.com/hmcts/div-case-orchestration-service/pull/259